### PR TITLE
MM-35071 - Update documentation for android e2e emulator setup

### PR DESCRIPTION
**Pull request by josephbaylon**
#### Summary
- Update documentation to reflect new setup for android emulator

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-35071

Related PR: https://github.com/mattermost/mattermost-mobile/pull/5347

### CODE REVIEW

**Comment by enahum** [comment]
Why is this changing and why it is no longer done with the npm command?

**Comment by josephbaylon** [comment]
> Why is this changing and why it is no longer done with the npm command?

This is related to https://github.com/mattermost/mattermost-mobile/pull/5347. A more detailed explanation is in the summary of that PR.

File: site/content/contribute/mobile/e2e/android.md [hunk 1]
```diff
@@ -9,13 +9,12 @@ subsection: Mobile End-to-End Testing
...
+  - Select `Pixel 4 XL` and hit **Next** button.
+  - Select **Recommended > API Level 30** and hit **Next** button.
+  - Keep the default AVD name `Pixel 4 XL API 30` and hit **Finish** button.
 
-   ```
-   sdkmanager "system-images;android-30;google_apis;x86"
-   sdkmanager --licenses
-   ```
-2. Create the emulator using `npm run e2e:android-create-emulator` from the `/detox` folder. Android testing requires an emulator named `detox_emu_api_30` and the script helps to create it automatically.
+Note: Android testing requires an emulator named `Pixel_4_XL_API_30`. The AVD name `Pixel 4 XL API 30` corresponds to the actual emulator name `Pixel_4_XL_API_30`.
```
Comment (saturninoabril) [inline site/content/contribute/mobile/e2e/android.md:17]: Is there no workaround to automatically create virtual device with Pixel 4 XL?

Comment (saturninoabril) [reply site/content/contribute/mobile/e2e/android.md:17]: Okay, I just read your description in this PR - https://github.com/mattermost/mattermost-mobile/pull/5347

**Comment by josephbaylon** [comment]
@enahum @saturninoabril I'll close this for now as we explore more alternatives. Once we've exhausted it I'll create another PR. I'll also revert the related mm code changes

**Pull request closed without merging**